docs/_config.yml

```diff
@@ -0,0 +1,40 @@
+title: coreos/zincati
+description: Zincati documentation
+# Temporary baseurl
+baseurl: "/zincati"
+url: "https://coreos.github.io"
+# Comment above and use below for local development
+# url: "http://localhost:4000"
+permalink: /:title/
+
+remote_theme: pmarsceill/just-the-docs
```

Comment:
We should consider forking this into our own repo under the `coreos` organization.